components/tools/OmeroJava/test/integration/chown/PermissionsTest.java

```diff
@@ -19,11 +19,14 @@
 
 package integration.chown;
 
+import static omero.rtypes.rstring;
```

Comment:
I don't think we ever came with a "general" rule.
If we remove the static import, it might be good to synch the other classes. I could start a card and we can clean up as we go/review the tests.